source/_integrations/alexa.intent.markdown

```diff
@@ -193,7 +193,17 @@ Custom slot type for scene support.
 
 The names must exactly match the scene names (minus underscores - Amazon discards them anyway and we later map them back in with the template).
 
-In the new Alexa Skills Kit, you can also create synonyms for slot type values, which can be used in place of the base value in utterances. Synonyms will be replaced with their associated slot value in the intent request sent to the Alexa API endpoint, but only if there are not multiple synonym matches. Otherwise, the value of the synonym that was spoken will be used.
+In the new Alexa Skills Kit, you can also create synonyms for slot type values, which can be used in place of the base value in utterances. Synonyms will be replaced with their associated slot value in the intent request sent to the Alexa API endpoint, but only if there are not multiple synonym matches (Deprecated). Otherwise, the value of the synonym that was spoken will be used.
+ 
+<div class='note'>
+
+In Home Assistant 2023.7.0 the synonym logic is going to change.
+Whenever there are synonym matches, the first match will be used.
+Only when there aren't any synonym matches, the value that was spoken will be used.
+  
+</div>
```

Comment:
We don't keep a changelog in our documentation. The documentation should reflect the current state.

Reply:
Ah okay, sorry for that.
I'm changing it.